app/experimenter/templates/experiments/radio_widget_closer.html

```diff
@@ -0,0 +1,7 @@
+{% for group, options, index in widget.optgroups %}
+  {% for option in options %}
+		<div class="radio ml-4">
```

Comment:
The spacing looks a bit odd here -- you have tabs instead of spaces I think.

Reply:
this is odd.. not sure what happened.

Reply:
Oh yep those are tab characters, definitely switch those to spaces.  Tabs are the devil 👿